mlir/test/Dialect/Rock/integration/blockwise_fill/lit.local.cfg

```diff
@@ -0,0 +1,3 @@
+# These tests are disabled in the interest of CI load
+# However, they should be passing.
+config.unsupported = True
```

Comment:
Won't it throw people that these tests are always unsupported? Wouldn't it be better to remove them?